alica_engine/src/engine/teammanager/TeamManager.cpp

```diff
@@ -84,12 +84,24 @@ void TeamManager::readSelfFromConfig()
 {
     essentials::SystemConfig& sc = essentials::SystemConfig::getInstance();
     const std::string localAgentName = _engine->getRobotName();
-    int id = sc["Local"]->tryGet<int>(-1, "Local", "ID", NULL);
-    if (id != -1) {
+    
+    constexpr auto notAValidID = (unsigned long long)(-1);
```

Comment:
uint64_t　is the cleaner choice imo.
also c-style casts?
I'd recommend clang-tidy 
/ramble off ;)